assets/css/edd-admin.css

```diff
@@ -1921,17 +1921,15 @@ body.download_page_edd-reports {
 }
 
 .edd-chip {
-	font-size: 7px;
+	font-size: 10px;
 	text-transform: uppercase;
 	font-weight: bold;
-	line-height: 8px;
+	line-height: 1;
 	vertical-align: middle;
 	padding: 2px 2px 3px 3px;
```

Comment:
Changing the padding to be consistent on all sides may resolve the issue @cklosowski mentioned; I think the `vertical-align` may also be contributing to the issue.

Reply:
With `vertical-align` (current):
![image](https://user-images.githubusercontent.com/18077352/79481212-060c2c00-7fc4-11ea-98b7-70a390d8bbc7.png)

Without `vertical-align`: 
![image](https://user-images.githubusercontent.com/18077352/79481008-c6454480-7fc3-11ea-8d2f-15e9491e0594.png)

Thoughts @robincornett and @cklosowski?